meta/labs_program/README.md

```diff
@@ -3,6 +3,11 @@
 This document will contain a brief overview of our labs program and what it'll
 entail going into the 2015 fall semester.
 
+The goal of the labs program is to bring the club playbook to the point where
+we're confident enough to start new clubs with it. We'll be doing this through
```

Comment:
Ahh, you got me. I'll get on fixing this.

On Wed, Aug 19, 2015 at 9:21 AM, Max Wofford <notifications@github.com>
wrote:

> In meta/labs_program/README.md
> <https://github.com/hackedu/hackedu/pull/61#discussion_r37434604>:
>
> > @@ -3,6 +3,11 @@
> >  This document will contain a brief overview of our labs program and what it'll
> >  entail going into the 2015 fall semester.
> >
> > +The goal of the labs program is to bring the club playbook to the point where
> > +we're confident enough to start new clubs with it. We'll be doing this through
>
> We'll be doing this through
> We'll be doing this *by*
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackedu/hackedu/pull/61/files#r37434604>.
>



-- 
Zach Latta
hackEDU | zach@hackedu.us